src/app/[room]/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { socket } from '@/socket'
+import SocketIoTestSection from '@/components/socketIoTest/SocketIoTestSection'
+import StoryPoints from '@/components/StoryPoints'
+import CardsContainer from '@/components/CardsContainer'
+
+export type ChatEvent = {
+	message: string
+	timeStamp: string
+	room: string
+}
+
+export default function UserRooms({ params }: { params: { room: string } }) {
+	const [roomId, setRoomId] = useState('')
+	const [userName, setUserName] = useState('')
+	// const [chatEvents, setChatEvents] = useState<ChatEvent[]>([])
+
+	// biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
+	useEffect(() => {
+		const { room } = params
+
+		console.log('%c>>> on user page load roomId:', 'color: red', room)
+
+		const user = 'Bob'
+
+		socket.emit('join-room', room, user)
+
+		setRoomId(room)
+		setUserName(user)
+	}, [])
+
+	return (
+		<main className='px-16 py-12 flex flex-col items-center gap-12 w-full'>
+			<h1 className='text-3xl text-green-400'>USER Scrum Diving Room</h1>
+			<div className='text-2xl text-zinc-500 h-[60vh] w-full flex flex-col justify-start items-center gap-8'>
+				<div className='text-2xl text-zinc-500 h-full w-full flex flex-row gap-8 justify-start items-center'>
+					<StoryPoints />
+					<CardsContainer />
+				</div>
+			</div>
+
+			{/*** Socket.io Test Section ***/}
```

Comment:
thought: I love [this type](https://preview.redd.it/iuy9fxt300811.png?auto=webp&s=156f5ffebbea5b30189eb36012e5bc7e3369c381) of comments (sarcastic :p ). Code comments should explain why something exists, not what it does.

Reply:
I like delineating sections of my code so I can better visually scan the document. My eye catches the comment with *** and I know where I am in a bit of code. In fact, I often make them bigger. This originally wasn't a component, it was a few dozen of lines of JSX.

Reply:
Ah I see, so this is not a normal code comment, it's a code comment to separate the areas of code. All good then ^.^

Reply:
DONE